.conf/dps_27/nginx.tasmoadmin.conf

```diff
@@ -0,0 +1,42 @@
+location ^~ /tasmoadmin {
+	# Manually deny some paths to ensure Tasmoadmin security
+	location ~* ^/tasmoadmin/(?:\.|data) {
+		deny all;
+	}
+
+	# Enable URL rewrites, required for sync clients
+	try_files $uri $uri/ /pydio/index.php?$args;
```

Comment:
@svh1985 
This needs to be removed as it would override the below rewrite rules as well?